app.config.js

```diff
@@ -3,19 +3,19 @@ import { secrets } from './src/config/secrets';
 export default ({ config }) => {
   const namespace = config.slug;
   const sentryApi = secrets[namespace].sentryApi;
-  const hooks = sentryApi?.config
-    ? {
-        postPublish: [
-          {
-            file: 'sentry-expo/upload-sourcemaps',
-            config: sentryApi.config
-          }
+
+  const plugins = sentryApi.config
+    ? [
+        ...config.plugins,
+        [
+          '@sentry/react-native/expo',
+          { project: sentryApi.project, organization: sentryApi.organization }
         ]
-      }
-    : {};
+      ]
+    : [...config.plugins];
```

Comment:
you do not need to spread all values of `config.plugins` creating a new array here. instead using `config.plugins` directly would be fine. if there is nothing to add, take it as is.